src/test/integration/table/updateTable.ts

```diff
@@ -0,0 +1,46 @@
+import { AssessmentController } from '@server/controller/assessment'
+import { MetadataController } from '@server/controller/metadata'
+import { UserController } from '@server/controller/user'
+
+import { assessmentParams } from '@test/integration/mock/assessment'
+import { tableParams } from '@test/integration/mock/table'
+import { userMockTest } from '@test/integration/mock/user'
+
+// test to update table
+export default () =>
+  test('Expect table to be updated', async () => {
+    const user = await UserController.getOne({
+      email: userMockTest.email,
+    })
+
+    const assessment = await AssessmentController.getOne({
+      assessmentName: assessmentParams.props.name,
+    })
+
+    const table = await MetadataController.createTable({
+      assessment,
+      user,
+      table: tableParams,
+    })
+
+    const tableUpdated = await MetadataController.updateTable({
+      assessment,
+      user,
+      table: {
+        ...table,
+        props: {
+          ...table.props,
+          name: 'updated name',
+        },
+      },
+    })
```

Comment:
Tests are good, but I would prefer to have a single place for the mock data used, eg 'mocks/table' or similar

What do you think?